apps/frontend/src/app/calendarPage/page.tsx

```diff
@@ -0,0 +1,129 @@
+'use client';
+
+import React, { useEffect, useState } from 'react';
+
+import { styles } from '@/components/calendarStyles';
+
+// Segédfüggvények
+const daysInWeek = ['Hétfő', 'Kedd', 'Szerda', 'Csütörtök', 'Péntek', 'Szombat', 'Vasárnap'];
+const shortDaysInWeek = ['H', 'K', 'Sz', 'Cs', 'P', 'Sz', 'V'];
+
+const monthNames = [
+  'Január',
+  'Február',
+  'Március',
+  'Április',
+  'Május',
+  'Június',
+  'Július',
+  'Augusztus',
+  'Szeptember',
+  'Október',
+  'November',
+  'December',
+];
+
+const getDaysInMonth = (month: number, year: number) => {
+  return new Date(year, month + 1, 0).getDate();
+};
+
+const getFirstDayOfMonth = (month: number, year: number) => {
+  const day = new Date(year, month, 1).getDay();
+  return day === 0 ? 7 : day;
+};
+
+// Calendar komponens
+const Calendar: React.FC = () => {
+  const [currentMonth, setCurrentMonth] = useState(7); // Augusztus
```

Comment:
This should be dynamic as well, we don't want to manually update the month and the year every time it changes :D
Consider using something like `const currentDate = new date()` and please get the month and year from this one